tests/test_utils.py

```diff
@@ -32,22 +31,12 @@ def test_get_default_timeout_margin(monkeypatch):
 
 
 def test_get_nearest_collector(monkeypatch):
-    monkeypatch.setitem(os.environ, constants.AWS_REGION, 'us-west-2')
-    collector = utils.get_nearest_collector()
-    assert collector == "api.thundra.io"
-
-    monkeypatch.setitem(os.environ, constants.AWS_REGION, 'us-east-2')
-    collector = utils.get_nearest_collector()
-    assert collector == "api-us-east-1.thundra.io"
-
-    monkeypatch.setitem(os.environ, constants.AWS_REGION, 'eu-west-1')
-    collector = utils.get_nearest_collector()
-    assert collector == "api-eu-west-1.thundra.io"
-
-    monkeypatch.setitem(os.environ, constants.AWS_REGION, 'eu-west-2')
-    collector = utils.get_nearest_collector()
-    assert collector == "api-eu-west-2.thundra.io"
-
-    monkeypatch.setitem(os.environ, constants.AWS_REGION, 'ap-')
-    collector = utils.get_nearest_collector()
-    assert collector == "api-ap-northeast-1.thundra.io"
+    regions = [ 'us-west-2', 'us-west-1', 'us-east-2', 'us-east-1', 'ca-central-1', 'sa-east-1',
+            'eu-central-1', 'eu-west-1', 'eu-west-2', 'eu-west-3', 'eu-north-1', 'eu-south-1',
+            'ap-south-1', 'ap-northeast-1', 'ap-northeast-2', 'ap-southeast-1', 'ap-southeast-2', 
+            'ap-east-1', 'af-south-1', 'me-south-1']
+
+    for region in regions:
+        monkeypatch.setitem(os.environ, constants.AWS_REGION, region)
+        collector = utils.get_nearest_collector()
+        assert collector == "{}.collector.thundra.io".format(region)
```

Comment:
Can you also add a test case to verify whether it uses `collector.thundra.io` when there is no region info available